pkg/controller/dnsendpoint/dnsendpoint_controller_test.go

```diff
@@ -424,7 +422,7 @@ func TestDNSEndpointReconcile(t *testing.T) {
 
 			cut := &ReconcileDNSEndpoint{
 				Client: fakeClient,
-				scheme: scheme.Scheme,
+				scheme: scheme,
```

Comment:
Tech debt: this field isn't actually used anywhere, and I imagine we have similar cases elsewhere. We should scrub such things.

This need not be tackled as part of the revendor, unless you want to, but please open a card.

Reply:
Reviewing again, I see this is probably the case in a lot of our reconciler structs.